plugins/mime/README.md

```diff
@@ -1,3 +1,20 @@
+## :warning: Not needed anymore
+
+As `image/jxl` is now supported by [shared-mine-info 2.2](https://gitlab.freedesktop.org/xdg/shared-mime-info/-/releases/2.2), it should not be necessary anymore to install this plugin.
```

Comment:
You probably meant `shared-mime-info 2.2`...

Reply:
That'd make for an interesting package name :)